subprojects/core/src/test/groovy/org/gradle/api/internal/tasks/options/OptionReaderTest.groovy

```diff
@@ -686,30 +794,45 @@ class OptionReaderTest extends Specification {
 
     public static class TestClassWithPropertyField {
         @Option(option = 'customOptionName', description = "custom description")
-        final Property<String> field1
+        final Property<String> fieldA
 
-        @Option(description = "Descr Field2")
-        final Property<String> field2
+        @Option(description = "Descr FieldB")
+        final Property<String> fieldB
 
-        @Option(description = "Descr Field3")
-        final Property<TestEnum> field3
+        @Option(description = "Descr FieldC")
+        final Property<TestEnum> fieldC
 
-        @Option(description = "Descr Field4")
-        final Property<Boolean> field4
+        @Option(description = "Descr FieldD")
+        final Property<Boolean> fieldD
 
-        @Option(description = "Descr Field5")
-        final ListProperty<String> field5
+        @Option(description = "Descr FieldE")
+        final Property<Integer> fieldE
 
-        @Option(description = "Descr Field6")
-        final ListProperty<TestEnum> field6
+        @Option(description = "Descr FieldF")
+        final ListProperty<Integer> fieldF
 
-        @Option(description = "Descr Field7")
-        final SetProperty<String> field7
+        @Option(description = "Descr FieldG")
+        final ListProperty<String> fieldG
 
-        @Option(description = "Descr Field8")
-        final SetProperty<TestEnum> field8
+        @Option(description = "Descr FieldH")
+        final ListProperty<TestEnum> fieldH
```

Comment:
💭 I guess the indexed system is what causes these hard-to-read diffs, too, right? With proper names Git would have a better time to figure out what has actually changed.